src/components/card/card.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import './card.scss';
+
+class Card extends Component {
+
+  checkAvailability() {
+    if (this.props.tutor.available) {
+      return (<button className="button-primary" onClick={this.props.chatNow}>chat now</button>)
+    } else {
+      return (<button className="button-primary" disabled onClick={this.props.chatNow}>sorry offline</button>)
+    }
+  }
+
+  msToTime() {
```

Comment:
Generic, it doesn't belong here. Create a `Time` class as a helper in `lib` folder in the root.